gdpr/article_32_security_of_processing.sp

```diff
@@ -0,0 +1,40 @@
+benchmark "article_32_security_of_processing" {
```

Comment:
```suggestion
benchmark "article_32" {
```

Do we need the full article name in here too? I think we can shorten to just the number instead